hearth/sig/lib/hearth/plugin_list.rbs

```diff
@@ -1,15 +1,15 @@
 module Hearth
-  class PluginList[T]
-    def initialize: (?Array[_Plugin[T]] plugins) -> void
+  class PluginList
```

Comment:
PluginLists for a given service must have plugins that operate on the service's Config object, so I think this should also be parameterized to specify that constraint.